x-pack/plugins/uptime/public/state/actions/monitor_management.ts

```diff
@@ -25,3 +26,7 @@ export const getServiceLocationsSuccess = createAction<ServiceLocations>(
   'GET_SERVICE_LOCATIONS_LIST_SUCCESS'
 );
 export const getServiceLocationsFailure = createAction<Error>('GET_SERVICE_LOCATIONS_LIST_FAILURE');
+
+export const getSyntheticsServiceEnabled = createAsyncAction<void, { serviceEnabled: boolean }>(
```

Comment:
I think we need to sync and quickly strategize on terms, since we both have features related to "enablement". The shared terms could get confusing in the codebase. How can we separate the concepts of public beta enablement vs admin enablement?

perhaps we can name this function `getSyntheticsServiceBetaEnabled` and add a comment? And perhaps rename variables like `isEnabled` to `isBetaEnabled`?

After tech preview, this code will be pulled out of the code base, correct?

Reply:
i have renamed everything from `enabled` to `allowed`.